torch_xla/csrc/runtime/pjrt_computation_client.cc

```diff
@@ -131,10 +109,15 @@ PjRtComputationClient::PjRtComputationClient() {
     TF_VLOG(1) << "Initializing PjRt GPU client...";
     bool async = sys_util::GetEnvBool(env::kEnvPjrtAsyncGpuClient, true);
     int local_process_rank = sys_util::GetEnvInt(env::kEnvPjRtLocalRank, 0);
-
     int global_process_rank = sys_util::GetEnvInt("RANK", local_process_rank);
-    auto distributed_client =
-        MaybeInitializeDistributedRuntimeClient(global_process_rank);
+    const std::string default_coordinator_port = "8547";
```

Comment:
Can we make this a static const member of DistributedRuntime?

Reply:
done